assets/styles/_dark-theme.scss

```diff
@@ -0,0 +1,106 @@
+body.dark-theme {
+
+  $dark: darken($brand, 40);
```

Comment:
suggestion: deep black `#03070b`  tends actually strain your eyes more. Example would be the MacOS dark theme, which is considered a pretty good dark theme, versus the Windows 10 dark theme implementation, which is not so good. Maybe use a lighter dark gray instead similar to what the MacOS mail app uses `#1E1E1E` for backgrounds.

Reply:
Thank you for he feedback, I'll change it ASAP and keep it in mind.